src/design-system/docs/hooks/useSourceFromExample.ts

```diff
@@ -0,0 +1,20 @@
+import { DocsExample } from '../types';
+
+export const useSourceFromExample = ({
```

Comment:
This isn't actually a hook, so maybe should be `sourceFromExample`?

Reply:
Once upon a time it was 😅 thanks for picking this up!